pkgs/applications/virtualization/colima/default.nix

```diff
@@ -35,7 +35,7 @@ buildGoModule rec {
     description = "Container runtimes on MacOS with minimal setup";
     homepage = "https://github.com/abiosoft/colima";
     license = licenses.mit;
-    platforms = platforms.darwin;
+    platforms = with platforms; darwin ++ linux;
```

Comment:
It can be removed, it is the default for `buildGoModule`.
```suggestion
```

Reply:
done